service-front/assets/js/moj/moj.modules/moj.sticky-nav.js

```diff
@@ -1,44 +0,0 @@
-// Dependencies: moj, _, jQuery
-
-(function () {
-  'use strict';
-
-  moj.Modules.StickyNav = {
-
-    init: function () {
-      _.bindAll(this, 'stickIt');
-      this.stickIt();
-    },
-
-    stickIt: function () {
-      if ($('nav.progress-indicator').length) {
```

Comment:
There are no HTML elements with the class progress-indicator, so this function will do nothing.